clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoConfig.java

```diff
@@ -94,6 +95,15 @@ public void read() {
     if (prop.containsKey(authKey)) {
       authentication = prop.getProperty(authKey);
     }
+
+    if (authKey.equals("oauth")) {
```

Comment:
Can this use "OAUTH_AUTH" to replace "oauth"?

Reply:
I didn't want this code to depend on Command as OAUTH_AUTH is private. I could extract to another class if you want.

Reply:
Never mind, it is just a minor issue.